packages/schema/src/schema/any.json

```diff
@@ -2,12 +2,16 @@
   "type": "object",
   "description": "Part::Any",
   "title": "IAny",
-  "required": ["Shape"],
+  "required": ["Content", "Type"],
   "additionalProperties": false,
   "properties": {
-    "Shape": {
+    "Content": {
       "type": "string",
-      "description": "The BREP string of the shape"
+      "description": "The string content of the object"
+    },
+    "Type": {
+      "type": "string",
```

Comment:
I think an enum of supported type is better, it helps validate the JCAD file and generate the correct interface for `_loadObjectFile` function